ui/lib/kv/addon/components/page/secret/metadata/version-history.hbs

```diff
@@ -0,0 +1,97 @@
+<KvPageHeader @breadcrumbs={{@breadcrumbs}} @pageTitle={{@path}}>
+  <:tabLinks>
+    <LinkTo @route="secret.details" data-test-secrets-tab="Secret">Secret</LinkTo>
+    <LinkTo @route="secret.metadata.index" data-test-secrets-tab="Metadata">Metadata</LinkTo>
+    <LinkTo @route="secret.metadata.versions" data-test-secrets-tab="Version History">Version History</LinkTo>
+  </:tabLinks>
+</KvPageHeader>
+
+<Toolbar />
+{{#if @metadata.canReadMetadata}}
```

Comment:
user's can't even navigate to this route without this permission, right?

Reply:
Correct. They can't. This was just a in case they use the URL to navigate here.